### DISCUSSION TITLE
Wavelog and Log4Om

### DISCUSSION THREAD

**Discussion by romain78180** [wavelog/wavelog#179] [Q&A] [answered]
Hello,
I use Log4Om and I' want use Wavelog when I'm remote.
Is it possible to connect together or to import .adi automatically?

Thx

**Comment by HB9HIL** [comment]
Unfortunately we don't have an option for that.

But you could use Wavelog also @ home ;-)

**Reply by romain78180** [reply]
it's a shame

**Reply by HB9HIL** [reply]
As far as I know does Log4OM not offer any kind of UDP Broadcast to push ADIF data to another source. So nothing we can do about in Wavelog.

**Comment by DK5EW** [comment]
VQlog is able to interchange adi with WSJT/MSHV and others. I use this at remote working fine.

**Comment by int2001** [comment]
Hi @romain78180 
I tried logging Log4OM --> Wavelog this morning. It is working.
Try the latest Version of Log4OM and put the full URL to the API into the URL-Field there. e.g. `https://yourlog/index.php/api/qso` or  `https://yourlog/api/qso` (Depends on your wavelog-installation).
Don't forget to chose the right station-id and the API-Key.

Log4OM will push the QSOs to Wavelog when you exit Log4OM (Perhaps there's a setting which pushes the QSO instantly, in my test-case it posted the QSOs on closing log4om).

**Comment by HB9HIL** [comment, marked as answer]
Hello @romain78180 

I tested again and Log4OM seems to brought some updates at the 15th March 2024. Now it's possible to connect Log4OM with Wavelog. There are basically two options: 

1. Option: 
Like @int2001 already mentioned you can open Log4OM 
`Program Configuration > Software Integration > Applications > Cloudlog`
and type in `https://yourlog/index.php/api/qso`, your API key and station location id. 
With this options the qso are transfered all in once when closing Log4OM. So it's not live. 

2. Option (recommended):
Install WavelogGate and configure it with your Wavelog URL `https://yourlog/index.php`, your API Key and the Station Location ID. 
Now open Log4OM `Program Configuration > Software Integration > Connections`. Here you can configure a UDP OUTBOUND connection. 
Settings Location:  
<img width="158" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/42082402-a2df-4ebe-bbab-1e2dd664cba2">    

Use Port `2333`, Type in a Connection Name and choose `ADIF_MESSAGE`:  
<img width="451" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/fd5e6f24-d1db-430b-8941-6f4d95e0f8f7">  
This is the enabled connection:  
<img width="447" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/8241a58d-a817-4784-9b98-9455d0e478ad">

Make sure WavelogGate is running. Now you can log QSO's in Log4OM and they got instantly transfered to Wavelog. So you have your log live in Wavelog aswell. 

<img width="1621" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/b816a18a-9246-463f-a630-88498c729ed5">
<img width="1626" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/499b29e6-c90b-45cb-b2bc-2683092d12b1">

I hope this helps.

**Reply by romain78180** [reply]
It works ! Thx
